zone-service/src/main/java/org/cloudfoundry/identity/uaa/zone/ZoneController.java

```diff
@@ -38,7 +44,17 @@ public ZoneController(ZoneService zoneService) {
 
     @GetMapping
     public ResponseEntity<ZoneResponse> getZone(@NotBlank(message = MANDATORY_VALIDATION_MESSAGE) @RequestParam String name) {
-        return new ResponseEntity<>(zoneService.getZoneDetails(name), HttpStatus.ACCEPTED);
+        return new ResponseEntity<>(zoneService.getZoneDetails(name), HttpStatus.OK);
+    }
+
+    @DeleteMapping
+    public ResponseEntity<?> deleteZone(@NotBlank(message = MANDATORY_VALIDATION_MESSAGE) @RequestParam String name) {
+        return zoneService.deleteZone(name);
```

Comment:
I guess we can have void return type in zoneService.deleteZone() and build response entity in controller something like ResponseEntity.status(HttpStatus.ACCEPTED).build()